### PR TITLE
KFServing v0.5 Announcement Blog

### PR DESCRIPTION
**Pull request by animeshsingh**

### CODE REVIEW

**Comment by animeshsingh** [comment]
@jbottum who approves blogs? Why is there only one OWNER, which doesn't make sense? 

cc @Bobgy

File: _posts/2020-02-23-kfserving-0.5.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+image: images/logo.png
+hide: false
+categories: [release, official]
+---
+
+#### By **Animesh Singh** and **Dan Sun**
+
+With inputs from : Yuzhui Liu, Tommy Li, Paul Vaneck, Andrew Butler, Srinivasan Parthasarathy
+
+[<u>According to Gartner</u>](https://www.gartner.com/document/3994810), "organizations are now interested not just in a “pathway” to production but in a “speedway” to production. However, most enterprises become dismayed when they find out how exceptionally challenging it is to operationalize AI, as opposed to how (deceptively) easy it is to perform proof of concepts (POCs) or small-scale projects."
```
Comment (kramachandran) [inline _posts/2020-02-23-kfserving-0.5.md:16]: Do we have clearance from Gartner to use this quote?

File: _posts/2020-02-23-kfserving-0.5.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+
+One key aspect of deploying models in production is being able to monitor the predictions for various metrics, and explaining the decisions the model is making. [<u>IDC predicts</u>](https://www.idc.com/getdoc.jsp?containerId=US46917020&pageType=PRINTFRIENDLY) by 2022, over 60% of consumer-focused AI Decisioning Systems in Finance, Healthcare, Government, and other regulated sectors will include provisions to explain their analysis and decisions. *Gartner states* that AI systems that have explainability as a standard feature will be expected.
```
Comment (kramachandran) [inline _posts/2020-02-23-kfserving-0.5.md:18]: Again, do we have clearance from Gartner to use this quote?

**Review by kramachandran — CHANGES_REQUESTED** [review]
Hi, generally quotes attributed to Gartner/Forester or any of the other Analyst firms need to be cleared with them. If you have an AR team, I'd suggest working with them to get clearance.

Alternatively, I don't think you really need these quotes to make the point your making. You could just make a similar point directly, without a quote.

File: _posts/2020-02-23-kfserving-0.5.md [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+comments: true
+image: images/logo.png
+hide: false
+categories: [release, official]
+---
+
+#### By **Animesh Singh** and **Dan Sun**
+
+With inputs from : Yuzhui Liu, Tommy Li, Paul Vaneck, Andrew Butler, Srinivasan Parthasarathy
+
```
Comment (theadactyl) [inline _posts/2020-02-23-kfserving-0.5.md:15]: Can we put at the front that this is from the KFServing Working Group? Would love to get WG attribution in blog posts.

**Review by theadactyl — CHANGES_REQUESTED** [review]
+1 to karthik's comments, otherwise one nit about attribution

File: _posts/2020-02-23-kfserving-0.5.md (hunk 1 above)
Comment (animeshsingh) [reply _posts/2020-02-23-kfserving-0.5.md]: while i have sent a note to Gartner and IDC to seek approval, for now I have removed that and rewrote the intro.

File: _posts/2020-02-23-kfserving-0.5.md (hunk 3 above)
Comment (animeshsingh) [reply _posts/2020-02-23-kfserving-0.5.md:15]: done

**Review by theadactyl — APPROVED** [review]
/approve /lgtm

**Pull request merged**